core/zms/src/main/rdl/Authority.rdli

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 Athenz Authors
```

Comment:
we no longer use year in copyright lines

Reply:
Fixed